src/test/java/donnafin/logic/parser/AddressBookParserTest.java

```diff
@@ -42,6 +43,11 @@ public void parseCommand_clear() throws Exception {
         assertTrue(parser.parseCommand(ClearCommand.COMMAND_WORD) instanceof ClearCommand);
     }
 
+    @Test
+    public void parseCommand_clearThrowsException_invalidInput() {
+        assertThrows(ParseException.class, () -> parser.parseCommand(ClearCommand.COMMAND_WORD, "25"));
```

Comment:
Could you also try other arguments like words and empty string:

e.g. `(ClearCommand.COMMAND_WORD, "")`, `(ClearCommand.COMMAND_WORD, "apple")`. These would be good equivalence partitions :)